src/components/Graph/Graph.js

```diff
@@ -4,53 +4,104 @@ import PropTypes from 'prop-types';
+    { id: 23, label: '2, Aristotle, History of Animals, 5.9, Pliny, Natural History 10.48', color: '##6d7771' }, //Aristotle: http://classics.mit.edu/Aristotle/history_anim.5.v.html, Pliny: http://data.perseus.org/citations/urn:cts:latinLit:phi0978.phi001.perseus-eng1:10.48
+    { id: 24, label: '1, Greek Anth. 7.212', color: '##6d7771' }, //Greek Anthology: https://anthologia.ecrituresnumeriques.ca/entities/314
+    { id: 25, label: '3, Greek Anth. 7.212, Antoninus Liberalis 14, Odyssey 5.333-353', color: '##6d7771' }, //Greek Anthology: https://anthologia.ecrituresnumeriques.ca/entities/314, Ant. Lib: https://en.wikipedia.org/wiki/Munichus, Homer: http://data.perseus.org/citations/urn:cts:greekLit:tlg0012.tlg002.perseus-eng1:5.313-5.364
+    { id: 26, label: '3, Antoninus Liberalis 14, Aelian, On the Nature of Animals, 1.1, Ovid. Met. 11.749-795', color: '##6d7771' }, //Ant. Lib: https://en.wikipedia.org/wiki/Munichus, Aelian:http://www.attalus.org/translate/animals1.html; Ovid: http://data.perseus.org/citations/urn:cts:latinLit:phi0959.phi006.perseus-eng1:11.749
+    { id: 27, label: '2, Moschus 3.42, Pausanias 10.31.6', color: '##6d7771' }, //Moschus: https://www.theoi.com/Text/Moschus.html#3, Pausanias: http://data.perseus.org/citations/urn:cts:greekLit:tlg0525.tlg001.perseus-eng1:10.31.6
+    
   ],
+  //these edges connect each attribute to the number of times it appears in the context
   edges: [
-    { to: 1, from: 7 },
-    { to: 1, from: 9 },
-    { to: 2, from: 9 },
-    { to: 3, from: 7 },
-    { to: 4, from: 7 },
-    { to: 5, from: 8 },
-    { to: 6, from: 9 },
+    { to: 9, from: 1 },
+    { to: 10, from: 2 },
+    { to: 11, from: 3 },
+    { to: 12, from: 4 },
+    { to: 13, from: 5 },
+    { to: 14, from: 6 },
+    { to: 15, from: 7 },
+    { to: 16, from: 8 },
+    
+    //these edges connect to the number of times each attribute appears in the context with the number of times it appears in common with another attribute
+    { to: 17, from: 9 },
+    { to: 17, from: 12 },
+    { to: 18, from: 10 },
+    { to: 18, from: 11 },
+    { to: 19, from: 11 },
+    { to: 19, from: 14 },
+ //   { to: 20, from: 10 }, I was mistaken in adding this node, there is no relationship between Memnon and woman
+ //   { to: 20, from: 13 },
+    { to: 21, from: 13 },
+    { to: 21, from: 14 },
+    { to: 22, from: 14 },
+    { to: 22, from: 15 },
+    { to: 23, from: 16 },
+    { to: 23, from: 15 },
+    { to: 24, from: 10 },
+    { to: 24, from: 14 },
+    { to: 25, from: 10 },
+    { to: 25, from: 15 },
+    { to: 26, from: 11 },
+    { to: 26, from: 15 },
+    { to: 27, from: 13 },
+    { to: 27, from: 15 },
+    
   ],
 };
 
+//const divinity = { i'm commenting these out because I don't think they're relevant to the layout of the graph I made now, but leaving them in the file for safety
```

Comment:
We can delete the `const divinity` here and everything will still work, but we need to delete the reference to the variable on line 116 below:

```
const data = {
  characteristics,
  divinity,
};
```